.gitignore

```
@@ -3,4 +3,5 @@ node_modules
 .vscode-test/
 *.vsix
 *.bin
-parsers/
\ No newline at end of file
+parsers/
+vendor/
```

Let's prefix both of these with a `/` so that we don't inadvertently ignore files vendored into `src/`, which I've had occasion to do before
```suggestion
/parsers/
/vendor/
```